src/geo/transform.js

```diff
@@ -201,7 +205,8 @@ class Transform {
     }
 
     get cameraWorldSize(): number {
-        return this._worldSizeFromZoom(this._zoomFromMercatorZ(this._camera.getDistanceToSeaLevel()));
+        const distance = Math.max(this._camera.getDistanceToElevation(this._averageElevation), 1e-20);
```

Comment:
why not 0 instead e-2?0

Reply:
I've replaced this with `Number.EPSILON`—even though it's a meaningless use of epsilon beyond indicating "a small number".

Maybe it would be appropriate to anchor this to a meaningful zoom level, in case you accidentally compute the distance to a point behind the camera.